frida_mode/src/prefetch.c

```diff
@@ -299,12 +300,15 @@ void prefetch_init(void) {
   /*
    * Configure the shared memory region to be removed once the process dies.
    */
+  #ifdef __ANDROID__
```

Comment:
Ah! I forgot about this part. Its not solved. I just verified it will still cause crash:
![grafik](https://user-images.githubusercontent.com/15239745/175172305-9d5b7a58-d7af-4145-a128-ad548ab981a6.png)

Reply:
Looks like AFL++ itself uses this header to fix Android issues with shared memory. Should be able to add a similar header to FRIDA mode. Perhaps shared memory should be abstracted to a BSP layer rather than using the Linux APIs directly. Check here to see where it is used...

https://grep.app/search?q=shmctl&filter[repo][0]=AFLplusplus/AFLplusplus&filter[path][0]=frida_mode/